_run/kube/Makefile

```diff
@@ -17,8 +17,8 @@ provider-run:
 provider-status:
 	curl -s "$(GATEWAY_ENDPOINT)/status" | jq
 
-.PHONY: provider-send-manifest
-provider-send-manifest:
+.PHONY: deployment-send-manifest
+deployment-send-manifest:
```

Comment:
Just to reduce overloading "deployment", perhaps `send-manifest-to-provider`, or keeping it short `send-manifest` imparts the action and subject of transfer.

Reply:
I'll go with `send-manifest`; good call.

The command eventually needs to send the manifest to all providers of a deployment, hence `deployment-`.  There's a few of these that are hard calls tho.